### PR TITLE
[MIRROR] Vore Panel and Character Setup HUD buttons

### PR DESCRIPTION
**Pull request by CHOMPStationBot**
Original PR: https://github.com/VOREStation/VOREStation/pull/11311
--------------------
dreamseeker_N2BO9DKgZy

The quadrant is complete.

I don't play this game, but I decided to like a couple days ago or whatever, and it was annoying to peck around for this verb to open the vore panel, so here's a button for it. Sprites (for each HUD style, I was just using eye-destroying orange for testing) by @ Very-Soft.

### CODE REVIEW

File: code/modules/vore/eating/living_vr.dm [hunk 1]
```diff
@@ -54,6 +55,19 @@
 	M.verbs += /mob/living/proc/vore_check_reagents //CHOMP If mob doesnt have bellies it cant use this verb for anything
 	M.verbs += /mob/living/proc/vore_bellyrub //CHOMP If mob doesnt have bellies it probably won't be needing this anyway
```
Comment (Arokha) [inline code/modules/vore/eating/living_vr.dm:56]: These would move to living/login in login.dm

File: code/modules/mob/living/login.dm [hunk 2]
```diff
@@ -24,5 +24,9 @@
 		verbs |= /mob/living/proc/vorebelly_printout
 		if(!vorePanel)
 			AddComponent(/datum/component/vore_panel)
+			
+	M.verbs += /mob/living/proc/vore_transfer_reagents //CHOMP If mob doesnt have bellies it cant use this verb for anything
+	M.verbs += /mob/living/proc/vore_check_reagents //CHOMP If mob doesnt have bellies it cant use this verb for anything
+	M.verbs += /mob/living/proc/vore_bellyrub //CHOMP If mob doesnt have bellies it probably won't be needing this anyway
```
Comment (Arokha) [inline code/modules/mob/living/login.dm:30]: Probably just verbs += (or |=). There's no M in this context and it won't compile

**Comment by Razgriz1032** [comment]
image
image

**Comment by Arokha** [comment]
Lots of devs don't play, funny enough

**Pull request merged**